tools/base/libboard/board_vc/board_vc.c

```diff
@@ -700,15 +766,14 @@ fpga_result print_phy_info(fpga_token token)
 
 		if (phy_info_array[0].phy_num == 4) {
 			switch (fpga_mode) {
-			case 1: /* 4x25g */
+			case FPGA_PHYGROUP_MODE_4_25G: /* 4x25g */
 				mask = 0xf;
```

Comment:
Redundant if we're falling through.
```C
				mask = 0xf;
```
You can get rid of line 770 since we are falling through to line 773 and doing the same operation.

Reply:
fixed